src/App.jsx

```diff
@@ -14,70 +15,98 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {(selectedGood === '')
+        && (
+          <h1 className="title is-flex is-align-items-center">
+            No goods selected
+          </h1>
+        )}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
+      {selectedGood &&
+        <h1 className="title is-flex is-align-items-center">
+          {selectedGood} is selected
+
+          {selectedGood && (
             <button
-              data-cy="AddButton"
+              data-cy="ClearButton"
               type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+              className="delete ml-3"
+              onClick={() => {
+                setGood('');
+              }}
+            />
+          )}
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+        </h1>
```

Comment:
you can replace all the upper logic with following
```suggestion
       <h1 className="title is-flex is-align-items-center">
       {selectedGood ? (
       <>
          {selectedGood} is selected
            <button
              data-cy="ClearButton"
              type="button"
              className="delete ml-3"
              onClick={() => {
                setGood('');
              }}
            />
          </>
          )
          :
          'No goods selected'
         }

        </h1>
```
no need to duplicate h1 tag, u can just manipulate its' content depending on `selecedGood`